feature-libs/cart/saved-cart/components/saved-cart-form-dialog/saved-cart-form-dialog.component.html

```diff
@@ -177,6 +181,7 @@ <h3 class="cx-saved-cart-form-title">
             </button>
             <button
               (click)="saveOrEditCart(cart?.code)"
+              [disabled]="form.invalid || isLoading$ | async"
```

Comment:
wrap the isLoading and pipe within an () 

Angular is complaining as it does not know what to do with the case you have above
```
form.invalid || isLoading$ | async
```

You can view the error by clicking the save cart for later or the edit icon. It will throw it in the devtools console